lib/github_changelog_generator/fetcher.rb

```diff
@@ -15,15 +15,15 @@ class Fetcher
 
     def initialize(options = {})
       @options = options || {}
-      @user = @options[:user]
-      @project = @options[:project]
       @github_token = fetch_github_token
-      github_options = { per_page: PER_PAGE_NUMBER }
-      github_options[:oauth_token] = @github_token unless @github_token.nil?
-      github_options[:endpoint] = @options[:github_endpoint] unless @options[:github_endpoint].nil?
-      github_options[:site] = @options[:github_endpoint] unless @options[:github_site].nil?
-
-      @github = check_github_response { Github.new github_options }
+      @github_options = { per_page: PER_PAGE_NUMBER }
+      @github_options[:user] = @options[:user]
```

Comment:
thank you Skywinder 

    Pada Selasa, 5 Januari 2016 2:15, Petr Korolev <notifications@github.com> menulis:
 

 In lib/github_changelog_generator/fetcher.rb:>        @github_token = fetch_github_token
> -      github_options = { per_page: PER_PAGE_NUMBER }
> -      github_options[:oauth_token] = @github_token unless @github_token.nil?
> -      github_options[:endpoint] = @options[:github_endpoint] unless @options[:github_endpoint].nil?
> -      github_options[:site] = @options[:github_endpoint] unless @options[:github_site].nil?
> -
> -      @github = check_github_response { Github.new github_options }
> +      @github_options = { per_page: PER_PAGE_NUMBER }
> +      @github_options[:user] = @options[:user]
Oh, yes. Miss it. I thought it was fixed already. 
Thanks!  
I will cover this options by tests later to prevent such similar in future. —
Reply to this email directly or view it on GitHub.